packages/design-system-tokens/src/themes/cmsgov-component.ts

```diff
@@ -449,11 +449,20 @@ export const components: AnyTokenValues = {
 
   'usa-banner': {
     '__background-color':                         t.color['gray-lightest'],
-    '__color':                                    t.color['black'],
-    '-close__background-color':                   t.color['gray-light'],
-    '-action__color':                             t.color['primary-darker'],
-    '-lock-icon__color':                          t.color['success-light'],
-    '-heading__font-size':                        '0.75rem',
+    '__color':                                    t.color['base'],
+    '__max-width':                                t.global['site-max-width'],
```

Comment:
Adding a custom prop to allow themes to determine width of USA banner (it's a token called out on USWDS docs as well). This prop uses the value defined in `site-max-width` to determine how the banner will line up with other global components like a header.

Experiencing issues getting `t.global[]` to build. Error says the value is undefined, but the theme file does have a `site-max-width` value within the `global` const, so I'm not sure if there's another way to import this value?